core/setup.cfg

```diff
@@ -45,6 +45,8 @@ keywords =
     android
     windows
     winforms
+	unix-like
+	freebsd
```

Comment:
Indentation is inconsistent here. Also "Unix-like" is a near meaningless term. Android and macOS are both "unix like".